cardform/src/main/java/com/mercadolibre/android/cardform/data/model/response/initinscription/InscriptionModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.mercadolibre.android.cardform.data.model.response.initinscription
+
+data class InscriptionModel(
```

Comment:
these classes that are not internals are usually only used internally ?

Reply:
In the case of this class I don't know if it's only used internally.